content/docs/0.8.0/users-guide/default-disk-and-node-config.md

```diff
@@ -1,41 +1,78 @@
-# Default disks and node configuration
+---
+title: Configuring Defaults for Nodes and Disks
+weight: 1
+---
 
-## Overview
+_Available as of v0.8.1_
 
-This feature allows the user to customize the default disks and node configurations in Longhorn for newly added nodes **using Kubernetes label and annotation**, instead of using Longhorn API or UI.
+This feature allows the user to customize the default disks and node configurations in Longhorn for newly added nodes using Kubernetes labels and annotations instead of the Longhorn API or UI.
 
-This feature is available since v0.8.1.
+Customizing the default configuations for disks and nodes is useful for scaling the cluster because it eliminates the need to configure Longhorn manually for each new node if the node contains more than one disk, or if the disk configuration is different for new nodes.
 
-## Usage
+> **Note:** Longhorn will not keep the node labels or annotations in sync with the current Longhorn node disks or tags.
```

Comment:
Add something like: `Nor will Longhorn keep the node disks or tags in sync with the nodes labels or annotations after the default disks or tags have been created`

Reply:
updated